apps/hash-graph/lib/graph/src/api/rest/utoipa_typedef/subgraph/edges.rs

```diff
@@ -26,14 +26,16 @@ pub enum KnowledgeGraphOutwardEdges {
 }
 
 // WARNING: This MUST be kept up to date with the enum variants.
-//   We have to do this because utoipa doesn't understand serde untagged:
-//   https://github.com/juhaku/utoipa/issues/320
-impl ToSchema for KnowledgeGraphOutwardEdges {
-    fn schema() -> RefOr<Schema> {
-        OneOfBuilder::new()
-            .item(<OutwardEdge<KnowledgeGraphEdgeKind, EntityIdAndTimestamp>>::schema())
-            .item(<OutwardEdge<SharedEdgeKind, OntologyTypeEditionId>>::schema())
-            .into()
+//   We have to do this because utoipa does use `OutwardEdge` as the schema reference
```

Comment:
I don't understand what this means

Reply:
It's using `Ref::from_schema_name("OutwardEdge")` (not even `OutwardEdge::<_, _>::schema().0)`) to generate the schema, which results in a wrong schema.

Reply:
Sure: 3d9fa77ebce40e91105d16e9171c6b38d32f96cd